test/extended/baremetal/common.go

```diff
@@ -58,9 +58,26 @@ func skipIfUnsupportedPlatformOrConfig(oc *exutil.CLI, dc dynamic.Interface) {
 	}
 }
 
+func skipIfProvisioningNetworkSet(dc dynamic.Interface) {
+	g.By("checking Provisioning Network CIDR")
+
+	provisioningClient := provisioningClient(dc)
+	provisioningConfig, err := provisioningClient.Get(context.Background(), "provisioning-configuration", metav1.GetOptions{})
+	o.Expect(err).NotTo(o.HaveOccurred())
+
+	provisioningSpec, _, err := unstructured.NestedMap(provisioningConfig.UnstructuredContent(), "spec")
+	o.Expect(err).NotTo(o.HaveOccurred())
+
+	provisioningNetworkCIDR, _, err := unstructured.NestedString(provisioningSpec, "provisioningNetworkCIDR")
```

Comment:
Does `provisioningNetworkCIDR` always exist even if it is null?. If no, maybe also worth to check bool value?

Reply:
after checking again, it looks like the provisioningNetworkCIDR entry should be absent in case of a Disabled network, so we can just set the bool and not even the value

Reply:
I don't know is it meaningful `provisioningNetworkCIDR` can exist but it is empty